src/use-cases/addproject.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { expect, describe, it, beforeEach } from 'vitest'
+
+import { CreateProjectUseCase } from './addProject'
+import { InMemoryProjectRepository } from '../repositories/in-memory-db/inMemoryProjectRepository'
+import { InMemoryUserRepository } from '../repositories/in-memory-db/inMemoryUserRepository'
+
+let projectRepository: InMemoryProjectRepository
+let userRepository: InMemoryUserRepository
+let sut: CreateProjectUseCase
+
+describe('Create Project Use Case', () => {
+  beforeEach(() => {
+    projectRepository = new InMemoryProjectRepository()
+    userRepository = new InMemoryUserRepository()
+    sut = new CreateProjectUseCase(projectRepository)
+  })
+
+  it('should be able to create Project', async () => {
+    const user = await userRepository.findById(
```

Comment:
it's better create a new user as [this constructor ](https://github.com/MatheusSanchez/orange-back/blame/master/src/repositories/in-memory-db/inMemoryUserRepository.ts#L15) will be deleted later; 

Otherwise, this test will fail when that happens.